command.go

```diff
@@ -286,8 +285,7 @@ func (c *Command) HelpTemplate() string {
 		return c.parent.HelpTemplate()
 	} else {
 		return `{{with or .Long .Short }}{{. | trim}}{{end}}
-{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}
-`
```

Comment:
There are also two newlines between the usage template and the help template (one is the spurious leading newline, the other is the spurious trailing newline). Would be nice to get this down to one as well (by removing the spurious leading newline).